net-p2p/retroshare/retroshare-0.6.4.recipe

```diff
@@ -0,0 +1,136 @@
+SUMMARY="An open source application that offers secure chat and file sharing"
+DESCRIPTION="Retroshare creates encrypted connections to your friends. Nobody \
+can spy on you. Retroshare is completely decentralized. This means there are \
+no central servers. It is entirely Open-Source and free. There are no costs, \
+no ads and no Terms of Service."
+HOMEPAGE="http://retroshare.sourceforge.net/"
+COPYRIGHT="2006-2018, drbob"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/RetroShare/RetroShare/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="84355c0f3be5ec1dfa7253e327ea1254f76f47739c233cfb8d0983ebd1a61f4a"
+SOURCE_DIR="RetroShare-$portVersion"
+PATCHES="retroshare-$portVersion.patchset"
+ADDITIONAL_FILES="retroshare.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86 !x86_gcc2"
```

Comment:
```diff
-SECONDARY_ARCHITECTURES="x86 !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
```
(You can drop the `!x86_gcc2` in `SECONDARY_ARCHITECTURES` because we don't need to mention broken secondary architectures.)

Thanks, however, for having kept the `?x86` in `ARCHITECTURES`. That's cool :)